tests/common/fixtures/advanced_reboot.py

```diff
@@ -501,6 +502,10 @@ def tearDown(self):
 
         self.__runScript(['remove_ip.sh'], self.ptfhost)
 
+        if self.postRebootCheckScript:
+            logger.info('Run the post reboot check script')
+            self.__runScript([self.postRebootCheckScript], self.duthost)
```

Comment:
Where is the code that copies this script to the dut?

Reply:
It’s done here [advanced_reboot.py#L219](https://github.com/Azure/sonic-mgmt/blob/581944481768f8457366aaecfc99983022af3696/tests/common/fixtures/advanced_reboot.py#L219)